frontend/packages/console-shared/src/components/quick-search/QuickSearchContent.tsx

```diff
@@ -34,24 +33,26 @@ const QuickSearchContent: React.FC<QuickSearchContentProps> = ({
   onSelect,
   closeModal,
   limitItemCount,
+  onListChange,
 }) => {
   return (
     <Split className="odc-quick-search-content">
       <SplitItem
         className={cx('odc-quick-search-content__list', {
-          'odc-quick-search-content__list--overflow':
-            catalogItems.length >= MAX_CATALOG_ITEMS_SHOWN,
+          'odc-quick-search-content__list--overflow': catalogItems.length >= limitItemCount,
         })}
       >
         <QuickSearchList
```

Comment:
I wonder if we want to make this a fixed max width... not sure we gain a lot by adding more space. cc @bgliwa01 

After a certain size we can just stop growing the list width-wise and give all that space to the content. Thoughts?